pylot/control/mpc/mpc.py

```diff
@@ -0,0 +1,313 @@
+"""
+Author: Fangyu Wu, Edward Fang
+Email: fangyuwu@berkeley.edu, edward.fang@berkeley.edu
+The code is adapted from
+https://github.com/AtsushiSakai/PythonRobotics/tree/master/
+PathTracking/model_predictive_speed_and_steer_control
+"""
+
+import numpy as np
+import cvxpy
```

Comment:
We would need to add `cvxpy` to our requirements.

Reply:
Done